content/TrueCommand/TCGettingStarted/AdminAccounts.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Creating the Admin Account"
+description: "This article describes how to create the administrator account in TrueCommand."
+weight: 30
+tags:
```

Comment:
Just noticed the aliases is missing from the front matter. Because the source article covered so much of the content moved into other articles and you can only add the alias in one of the new articles, I suggest adding this here in this article because it is the starting pointr Add between the front matter weight and tags is a good place for it:
aliases: /truecommand/tcgettingstarted/install/

Reply:
Mic and I have reviewed.